test/links_test.rb

```diff
@@ -78,7 +78,23 @@
   end
 
   test 'qualified url surrounded by angled brackets' do
-    assert_xpath '//a[@href="http://asciidoc.org"][text()="http://asciidoc.org"]', convert_string('<http://asciidoc.org> is the project page for AsciiDoc.'), 1
+    assert_xpath '//a[@href="http://asciidoc.org"][@class="bare"][text()="http://asciidoc.org"]', convert_string('<http://asciidoc.org> is the project page for AsciiDoc.'), 1
+  end
+
+  test 'qualified url surrounded by angled brackets in unconstrained context' do
+    assert_xpath '//a[@href="http://asciidoc.org"][@class="bare"][text()="http://asciidoc.org"]', convert_string('URLは<http://asciidoc.org>。fin'), 1
+  end
+
+  test 'multiple qualified urls surrounded by angled brackets in unconstrained context' do
+    assert_xpath '//a[@href="http://asciidoc.org"][@class="bare"][text()="http://asciidoc.org"]', convert_string('URLは<http://asciidoc.org>。URLは<http://asciidoc.org>。'), 2
+  end
+
+  test 'qualified url surrounded by escaped angled brackets' do
+    assert_xpath '//p[text()="<http://asciidoc.org>"]', convert_string('\\<http://asciidoc.org>'), 1
```

Comment:
Yes, this test is correct. It is escaping the whole form. This is the behavior that AsciiDoc.py has. Asciidoctor actually gets this wrong currently as it leaves the backslash and still applies the autolink. This PR will fix that.

Asciidoctor also happens to support `<\https://asciidoc.org>` which will prevent the autolink and leave the angled brackets around the URL.

If you want an autolink that is surrounded by a pair of angled brackets, the syntax is `<<https://asciidoc.org>>` (it looks like an internal xref, but it's not because it gets matched first). That's also consistent with AsciiDoc.py.

Reply:
...or it can be written as `<https://asciidoc.org[]>`.